lib/getPackageReportData.js

```diff
@@ -53,9 +53,18 @@ function getPackageReportData(packageEntry, callback) {
 
 		var author = json.author
 		if (_.isObject(author)) {
-			author = author.name;
-			if (author.email) { author = author + ' ' + author.email; }
-			if (author.url) { author = author + ' ' + author.url; }
+			author = author.name || ""
```

Comment:
please use single ticks for strings to be consistent

Reply:
Ahh sorry.